azimuth_caas_operator/operator.py

```diff
@@ -115,11 +115,22 @@ async def cluster_type_updated(body, name, namespace, labels, **kwargs):
         await _update_cluster_type(K8S_CLIENT, name, namespace, cluster_type.status)
 
 
+@kopf.on.resume(registry.API_GROUP, "cluster")
+async def cluster_resume(body, name, namespace, **kwargs):
+    LOG.debug(f"Resuming cluster {name} in {namespace}")
+    # Ensure that the clusterID is set for all clusters
```

Comment:
Is it worth noting we only need this so we can fix up older appliances to work with valero, so we can hopefully remove this eventually?

Reply:
Yes - for clusters created after the change or for old clusters that have been updated, this becomes a no-op resume handler